### ISSUE TITLE
plot_cells warning : Adding missing grouping variables: `cell_group`, `cell_color` Joining, by = c("cell_group", "cell_color")

### ISSUE THREAD

**Issue by Sophia409** [cole-trapnell-lab/monocle3#110]
Hi,
First question,why it gave the warning that adding missing grouping variables after I plot_cells.I'm really confused.How to add it?
Second question, I have cluster cells and see the cluster in the picture.But why I couldn't find the monocle cluster information in the colData column?
```
>plot_cells(incds,color_cells_by="partition",group_cells_by="cluster",cell_size=1,alpha=1)+theme(aspect.ratio=1,plot.title = element_text(size = rel(2)))
No trajectory to plot. Has learn_graph() been called yet?
Adding missing grouping variables: `cell_group`, `cell_color`
Joining, by = c("cell_group", "cell_color")
> tail(colData(ppincds))
DataFrame with 6 rows and 9 columns
                    orig.ident nCount_RNA nFeature_RNA        time       percent.mito RNA_snn_res.0.5 seurat_clusters
                   <character>  <numeric>    <integer> <character>          <numeric>     <character>        <factor>
TTGTGGAGTACGTGAG SeuratProject      22079         5534          p7  0.017165632501472              23              23
TTTACTGTCTTGTGCC SeuratProject       2410         1367          p7 0.0278008298755187              23              23
TTTATGCGTAAGTAGT SeuratProject      13149         4447          p7 0.0233477831013765              23              23
TTTCCTCCATGACAAA SeuratProject       1786         1164          p7 0.0369540873460246              23              23
TTTGATCGTTTAGTCG SeuratProject      22556         5249          p7 0.0150292605071821              23              23
TTTGGTTGTATTCTCT SeuratProject      16328         5046          p7 0.0346031357177854              23              23
                 integrated_snn_res.0.5       Size_Factor
                               <factor>         <numeric>
TTGTGGAGTACGTGAG                     23 0.909301291891113
TTTACTGTCTTGTGCC                     23 0.617109360049496
TTTATGCGTAAGTAGT                     23 0.953714465531039
TTTCCTCCATGACAAA                     23 0.539970690043309
TTTGATCGTTTAGTCG                     23 0.776061770971336
TTTGGTTGTATTCTCT                     23 0.899951150072181

```

![图片](https://user-images.githubusercontent.com/30619010/59488775-007d1b80-8eb3-11e9-86a4-aa74a3be5957.png)

**Comment by hpliner** [comment]
The plot_cells issue has already been reported (#106) and we expect to release a fix in 0.1.1 which should come out next week.

Clusters are now accessible through the `clusters(cds, reduction_method = "UMAP"))` function

**Comment by hpliner** [comment]
Fixed in 0.1.1
